### DISCUSSION TITLE
出现错误：加载帮助xaml文件失败:意外的XML声明。

### DISCUSSION THREAD

**Discussion by AISophon** [Meloong-Git/PCL#2167] [提问]
![error](https://github.com/Hex-Dragon/PCL2/assets/130284780/c09f181a-5898-4e2a-ac1f-d4eb0d1f70b4)

这是加载的自定义帮助文件`address.xaml`中的所有内容：

`<?xml version="1.0" encoding="UTF-8"?>`

这是加载的自定义帮助文件`address.json`中的所有内容：

```
{
    "__Author__": "AISophon",
	"Title": "玛卡巴卡",
 	"Description": "None",
	"Types": ["Text"]
}
```

**Comment by Chiloven945** [comment]
请……请不要往标题里塞这么多内容，很影响观感的（……
还有就是发一下文件

**Reply by AISophon** [reply]
哦好的

蓝奏云链接：`https://wwwa.lanzouj.com/isBQP117ztgf`
